tests/route/test_route_perf.py

```diff
@@ -0,0 +1,117 @@
+import pytest
+import json
+from datetime import datetime
+from tests.common.utilities import wait_until
+
+pytestmark = [
+    pytest.mark.topology('any'),
+    pytest.mark.device_type('vs')
+]
+
+ROUTE_JSON = 'route.json'
```

Comment:
>ROUTE_JSON [](start = 0, length = 10)

Use ansible tempfile module to generate a safe temp filename?

Reply:
Using `mktemp` in DUT to generate this temp file.